packages/oats/src/generate-types.ts

```diff
@@ -417,9 +417,6 @@ export function run(options: Options) {
         const endpoint: oas.OperationObject = schema.paths[path][method];
         oautil.errorTag(`in ${method.toUpperCase()} ${path} query`, () =>
           response.push(
-            generateNamedTypeDefinitionDeclaration(
```

Comment:
do not need to generate separate assign and declaration for reflection types. Instead we generate a declaration and assign it in the same expression